Description: fix the load path of the extension
 The require of the extension in the Ruby library assumes it is in stalled
 as a gem and hence loads 'ox/ox', but it should just load the installed
 'ox.so'.
Author: Paul van Tilburg <paulvt@debian.org>
Last-Update: 2012-02-22

--- a/lib/ox.rb
+++ b/lib/ox.rb
@@ -99,4 +99,4 @@
 require 'ox/bag'
 require 'ox/sax'
 
-require 'ox/ox' # C extension
+require 'ox.so' # C extension
